sonar-csharp-plugin/src/main/java/org/sonar/plugins/csharp/CSharp.java

```diff
@@ -36,4 +36,8 @@ public CSharp(Settings settings) {
     return settings.getStringArray(CSharpPlugin.FILE_SUFFIXES_KEY);
   }
 
+  @Override
+  public boolean equals(Object o) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This class overrides "equals()" and should therefore also override "hashCode()". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1206)